guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -77,7 +83,7 @@ To import files from a file type repository in a local directory, complete the f
 . In the *Name* field, enter a name for the repository.
 {Project} automatically completes this field based on what you enter for *Name*.
 . From the *Type* list, select the content type of the repository.
-. In the *Upstream URL* field, enter the local directory with the repository to use as the source, in the form `\file:///__my_file_repo__`.
+. In the *Upstream URL* field, enter the local directory with the repository to use as the source, in the form `\http://sat_fqdn/pub/__new_repo__`.
```

Comment:
This line should not change, users are importing a local directory and using `file:///`

Reply:
Are these issues similar:
https://bugzilla.redhat.com/show_bug.cgi?id=2133574 and 
https://bugzilla.redhat.com/show_bug.cgi?id=2131854

Reply:
> Are these issues similar: https://bugzilla.redhat.com/show_bug.cgi?id=2133574 and https://bugzilla.redhat.com/show_bug.cgi?id=2131854

Yes, those look to be the same root cause as documente dhere.

Reply:
> > Are these issues similar: https://bugzilla.redhat.com/show_bug.cgi?id=2133574 and https://bugzilla.redhat.com/show_bug.cgi?id=2131854
> 
> Yes, those look to be the same root cause as documented here.

Should we make any additional update to cover https://bugzilla.redhat.com/show_bug.cgi?id=2133574?